lib/src/space/grounding.rs

```diff
@@ -330,13 +330,13 @@ impl PartialEq for GroundingSpace {
 
 impl Debug for GroundingSpace {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "GroundingSpace")
+        write!(f, "GroundingSpace-{:?}", self as *const GroundingSpace)
```

Comment:
Is the idea that a memory address should be printed for the GroundingSpace?

fmt has a markup for pointers.  https://doc.rust-lang.org/std/fmt/trait.Pointer.html

Reply:
@vsbogd , my approval means I'm agnostic wrt @luketpeterson comment.

Reply:
Thanks, Luke, applied suggestion in https://github.com/trueagi-io/hyperon-experimental/pull/436/commits/1fb6ee8d69d90894a15e80f1e8ae5b0fdd58bcbc